src/Compilers/CSharp/Portable/Operations/CSharpConversionExpression.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Semantics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal abstract class BaseCSharpConversionExpression : BaseConversionExpression<Conversion>
+    {
+        protected BaseCSharpConversionExpression(Conversion conversion, bool isExplicitInCode, bool throwsExceptionOnFailure, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+            base(conversion, isExplicitInCode, throwsExceptionOnFailure, syntax, type, constantValue)
+        {
+        }
+
+        public override string LanguageName => LanguageNames.CSharp;
+        // We override this here so that we don't return the method symbol if the internal conversion is a MethodGroup conversion, instead of
```

Comment:
sorry, my down is synonymous with "up" :)   Push this into the most base conversion expressoin type.